README.md

```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
 
 If you're passionate about something not listed above, please don't hesitate to submit your idea for consideration.
 
+## Resources
+Trello Board [https://trello.com/b/M9VUQl9S/nodejs-collection](https://trello.com/b/M9VUQl9S/nodejs-collection)
+
+### Key Roles:
+- **Contributor/Writer** : Person submitting article or idea for inclusion on the Node.js Collection
+- **Editor** : Technical Reviewer. Person who ensures the content's technical integrity.
+
 ## Guidelines/General Tips of Submissions
 
-- Posts from members and community members must be vendor neutral, though they may mention vendors involved in specific project deployments if this serves the purpose of educating readers. 
-- Posts about hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, are allowed, but again shouldn’t feel like an advertisement for your services or company. 
+- Posts from members and community members must be vendor-neutral, though they may mention vendors involved in specific project deployments if this serves the purpose of educating readers.
+- Posts about hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, are allowed, but again shouldn’t feel like an advertisement for your services or company.
```

Comment:
I'm not sure if this is actually valuable in practice. Have we had any posts that are like this?

Reply:
@bnb I didn't get what you mean? Can you explain a bit?
To me, "Advertisement for your services or company" is not the best. So, it'd be good to keep this line.